tests/test_model_manager.py

```diff
@@ -164,7 +164,7 @@ def test_train_save_dir_overwrite(
         "train_configuration.yaml",
         "test_metrics.json",
         "val_metrics.json",
-        "epoch=0-step=10.ckpt",
+        "epoch=0-step=11.ckpt",
```

Comment:
Caused by recent change in 1.6.0 (released 2022-03-29)
> The filename of checkpoints created with ModelCheckpoint(filename='{step}') is different compared to previous versions. A checkpoint saved after 1 step will be named step=1.ckpt instead of step=0.ckpt (https://github.com/PyTorchLightning/pytorch-lightning/pull/11805)